data/Cshiva773.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Shivansh Chauhan",
+  "type": "personal",
+  "bio": "I am an enthusiast in the field of software engineering and have a deep interest in problem-solving",
+  "links": [
++    {
++      "name": "Let's get together on Twitter",
++      "url": "https://twitter.com/greatshiva773",
++      "icon": "FaTwitter"
++    },
++    {
++      "name": "Find me on Github",
++      "url": "https://github.com/Cshiva773",
++      "icon": "FaGithubAlt"
++    },
++    {
++      "name": "Let's connect on Linkedin",
++      "url": "https://www.linkedin.com/in/shivansh-chauhan-8677b4228/",
++      "icon": "FaLinkedin"
++    }
```

Hey @Cshiva773, please remove these trailing `+` signs in the lines `6-20`.

```suggestion
    {
      "name": "Let's get together on Twitter",
      "url": "https://twitter.com/greatshiva773",
      "icon": "FaTwitter"
    },
    {
      "name": "Find me on Github",
      "url": "https://github.com/Cshiva773",
      "icon": "FaGithubAlt"
    },
    {
      "name": "Let's connect on Linkedin",
      "url": "https://www.linkedin.com/in/shivansh-chauhan-8677b4228/",
      "icon": "FaLinkedin"
    }
```